fact-bounty-client/src/shared/OAuthContainer/index.js

```diff
@@ -1,38 +1,14 @@
 import React from "react";
 
+import FacebookContainer from "./FacebookContainer";
+import GoogleContainer from "./GoogleContainer";
 import "./index.css";
 
-class OauthContainer extends React.Component {
```

Comment:
These were allowing this OAuth container to load asynchronously, which would other components to mount first. It would allow register page to load first

Reply:
@Anmolbansal1 Yes I know Oauth component was importing google and facebook container asynchronously but I feel It was not worth it because.
-  As there was 2 continuous await statement which might neutralize time saved by asynchronously loading (to improve this we can use Promise.all).
-  Moreover, Google and facebook container components are not so heavy so asynchronously loading will not have any significant impact. 
- Asynchronous loading also made Oauth a complex component.

So I decided to changed it into simple function component 😃